test/extensions/filters/network/common/fuzz/filter_fuzz_support.bzl

```diff
@@ -0,0 +1,36 @@
+load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
+
+def _selected_extension_target(target):
+    return target + "_envoy_extension"
+
+def fuzz_filters_targets(filters):
+    all_extensions = EXTENSIONS
+
+    return [v for k, v in all_extensions.items() if (k in filters)]
+
+def generate_from_template(**kwargs):
+    _generate_from_template(
+        source_file = "{name}.cc".format(**kwargs),
+        **kwargs
+    )
+
+def _generate_from_template_impl(ctx):
+    ctx.actions.expand_template(
+        template = ctx.file.template_file,
+        output = ctx.outputs.source_file,
+        substitutions = {
+            "{{FILTERS}}": "\"" + ("\", \"".join(ctx.attr.filters)) + "\"",
```

Comment:
I might be missing something, but why is the templating needed here?

Reply:
You probably don't miss anything here, but just have detected my deficiencies in bazel. What I took as a draft was using this style, so I stuck to it, because it worked.

I try to come up with something less elaborate.

Reply:
You are absolutely correct. This is superstitious and a left-over from the initial version of the change. I removed the templating.